content/projects/gsoc/2020/projects/git-plugin-performance.adoc

```diff
@@ -23,4 +22,41 @@ links:
   idea: /projects/gsoc/2020/project-ideas/git-plugin-performance
 ---
 
-// TODO: add details here, remove `showGoogleDoc: true`
+The project aims to improve overall performance of git plugins by applying micro-benchmarking principles to create a comparitive study of CLI and JGit implementations of "git" in the plugins. The comparitive study can be used to codify the optimal selection of git implementation for an operation. Furthermore, it aims to fix non-trivial issues related to redundancy at hot code paths in git plugin.
```

Comment:
It is OK to use a sentence per line and it makes it easier for reviewers to see changes during the review process. 
 No need to change this pull request, just for future reference.

Spelling fix.
```suggestion
The project aims to improve overall performance of git plugins by applying micro-benchmarking principles to create a comparative study of CLI and JGit implementations of "git" in the plugins.
The comparative study can be used to codify the optimal selection of git implementation for an operation.
Furthermore, it aims to fix non-trivial issues related to redundancy at hot code paths in git plugin.
```